docs/modules/trino/pages/index.adoc

```diff
@@ -1,55 +1,65 @@
 = Stackable Operator for Trino
-:description: The Stackable Operator for Trino is a Kubernetes operator that can manage Trino clusters. Learn about its features, resources, dependencies and demos, and see the list of supported Trino versions.
-:keywords: Stackable Operator, Trino, Kubernetes, k8s, operator, data science, data exploration
-
+:description: The Stackable operator for Trino is a Kubernetes operator that can manage Trino clusters. Learn about its features, resources, dependencies and demos, and see the list of supported Trino versions.
+:keywords: Stackable operator, Trino, Kubernetes, k8s, operator, data science, data exploration
+:trino: https://trino.io/
+:github: https://github.com/stackabletech/trino-operator/
+:crd: {crd-docs}/trino.stackable.tech/trinocluster/v1alpha1/
```

Comment:
do we want to link the catalog CRD as well?

Reply:
I think it's fine like this, the catalog CRD is also linked in multiple other places, I think once you start working with catalogs you will find it anyways. this is just to get people "onto the right track" so to say